README.md

```diff
@@ -30,3 +31,28 @@ function mystery(n) {
 Add your answer to this markdown file. [This
 page](https://docs.github.com/en/get-started/writing-on-github/working-with-advanced-formatting/writing-mathematical-expressions)
 might help with the notation for mathematical expressions.
+
+
+Recurrence relation for mystery():
+
+$T(n) =$ undefined if $n <= 1$ <br>
```

Comment:
How can it be undefined?

Reply:
Running the code as is when I pulled it down the function returns undefined when I console.log the output. Should I just make the base case 1?

Reply:
The recurrence relation tells you how much time something takes, not what it returns.